framework/synapse-framework-exception/src/main/resources/error-messages.properties

```diff
@@ -19,3 +19,4 @@ MISSING_HTTP_HEADER_ERROR=Missing required HTTP header: {0}.
 AUTHENTICATION_ERROR=Your client does not have permission to get this URL from this server.
 TOO_MANY_REQUESTS=Your client has issued too many requests.
 LOCKED=Your client has issued a request to a locked resource.
+CONFLICT=The resource you are trying to update is outdated. Please try again.
```

Comment:
I wouldn't call it CONFLICT either, I would call it 'RESOURCE_OUT_OF_SYNC_ERROR' I think, or something like that more explicit than just CONFLICT.

Reply:
Done